### PR TITLE
[API] added 'place' dummy interface for variable

### PR DESCRIPTION
**Pull request by feifei-111**
### PR types

Others
### PR changes

Others
### Describe

'place' interface is added to variable which exists in dygraph mode tensor

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/dygraph_to_static/test_place.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+        def func(x):
+            x = paddle.to_tensor([1, 2, 3, 4])
+            p = x.place()
+            print(p)
+            return x
+
+        x = paddle.to_tensor([3])
+        print(paddle.jit.to_static(func).code)
+        print(paddle.jit.to_static(func)(x))
```
Comment (Aurelius84) [inline python/paddle/fluid/tests/unittests/dygraph_to_static/test_place.py]: 单测规范要求必须要有self.assert语句

File: python/paddle/fluid/layers/math_op_patch.py [hunk 2]
```diff
@@ -147,6 +147,16 @@ def cuda(self):
...
         return self
 
+    @static_only
+    def place(self):
+        """
+            Varaibel should not have place() interface.
+            But this interface can greatly facilitate dy2static.
+            We give a warnning here.
+        """
+        warnings.warn("Variable dont have 'place' attribute!")
```
Comment (Aurelius84) [inline python/paddle/fluid/layers/math_op_patch.py]: 注意单测拼写错误问题。另外要注意报错文案的书写，要清晰，明确，可读

File: python/paddle/fluid/layers/math_op_patch.py [hunk 3]
```diff
@@ -147,6 +147,16 @@ def cuda(self):
         """
         return self
 
+    @static_only
+    def place(self):
+        """
+            Varaibel should not have place() interface.
+            But this interface can greatly facilitate dy2static.
```
Comment (Aurelius84) [inline python/paddle/fluid/layers/math_op_patch.py]: 注意单词拼写

**Review by Aurelius84 — APPROVED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by dingjiaweiww — APPROVED** [review]

**Pull request merged**